server/app/services/webhook_service.py

```
@@ -854,3 +858,345 @@ async def _send_webhook_to_client(
+        VAPI sends this event when an inbound call arrives and the phone number
+        has no assistantId configured. Must respond within 7.5 seconds with
+        assistant configuration.
+        
+        Args:
+            data: Webhook data from VAPI (contains call.customer, call.phoneNumber)
+            db: Database session
+            
+        Returns:
+            Dict with assistantId or assistant configuration and assistantOverrides
+        """
+        from app.models import Call, ServiceRecord, Organization, Rooftop
```

Ruff (0.14.7)

Redefinition of unused `ServiceRecord` from line 14

Remove definition: `ServiceRecord`

(F811)